source/installer.py

```diff
@@ -237,10 +238,53 @@ def removeOldProgramFiles(destPath):
 	"URLInfoAbout":versionInfo.url,
 }
 
-def registerInstallation(installDir,startMenuFolder,shouldCreateDesktopShortcut,startOnLogonScreen,configInLocalAppData=False):
-	with winreg.CreateKeyEx(winreg.HKEY_LOCAL_MACHINE,"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\NVDA",0,winreg.KEY_WRITE) as k:
-		for name,value in uninstallerRegInfo.items(): 
-			winreg.SetValueEx(k,name,None,winreg.REG_SZ,value.format(installDir=installDir))
+
+def getDirectorySize(path: str) -> int:
+	"""Calculates the size of a directory in bytes.
+	"""
+	total = 0
+	with os.scandir(path) as iterator:
+		for entry in iterator:
+			if entry.is_file():
+				total += entry.stat().st_size
+			elif entry.is_dir():
+				total += getDirectorySize(entry.path)
+	return total
+
+
+def registerInstallation(
+		installDir: str,
+		startMenuFolder: str,
+		shouldCreateDesktopShortcut: bool,
+		startOnLogonScreen: bool,
+		configInLocalAppData: bool = False
+):
+	calculatedUninstallerRegInfo = uninstallerRegInfo.copy()
+	# EstimatedSize is in KiB
+	calculatedUninstallerRegInfo.update(EstimatedSize=getDirectorySize(installDir) // 1024)

```

Comment:
Adding a debug line here may be helpful in future.

```suggestion
	# EstimatedSize is in KiB
	estimatedSize = getDirectorySize(installDir)
	log.debug(f"Estimated install size {estimatedSize}")
	calculatedUninstallerRegInfo.update(EstimatedSize=estimatedSize // 1024)
```